ui/components/AlertsTable.tsx

```diff
@@ -46,10 +49,20 @@ function AlertsTable({ className, rows = [] }: Props) {
     };
   }
 
+  const [yamlView, setYamlView] = React.useState<Alert>(null);
+
   const alertFields: Field[] = [
     {
       label: "Name",
-      value: "name",
+      value: (a) => (
+        <Text
+          onClick={() => setYamlView(a)}
+          color="primary10"
+          className="pointer"
```

Comment:
Passing from outside the class defined within a component is not done usually. The className is reserved for passing external styling/classes. The external components are not supposed to know anything about internals of the Text component. It's better to add a prop for this. Either an optional useHandPointer or smth. like that or an optional string prop to pass a cursor type and apply it within the component like:

`${(props) => props.cursor && `cursor:${props.cursor};`};`